common-npm-packages/webdeployment-common-v2/deployusingmsdeploy.d.ts

```diff
@@ -0,0 +1,17 @@
+import { WebDeployArguments, WebDeployResult } from './msdeployutility';
```

Comment:
I'm not sure that we should commit these `d.ts` files.

Reply:
done